test/Passes/tpp-identity.mlir

```diff
@@ -0,0 +1,63 @@
+// RUN:  tpp-opt %s  --default-tpp-passes | tpp-run -e entry  -entry-point-result=void | FileCheck %s
+
+func.func @pack1(%in: tensor<4x4xf32>, %out: tensor<2x2x2x2xf32>) ->  tensor<2x2x2x2xf32> {
+  %1 = tensor.pack %in inner_dims_pos = [0, 1] inner_tiles = [2,2] into %out : tensor<4x4xf32> -> tensor<2x2x2x2xf32>
+  return %1 : tensor<2x2x2x2xf32>
+}
+
+func.func @pack2(%0: tensor<1x2x2x4xf32>, %1:  tensor<1x2x2x2x2xf32>)-> tensor<1x2x2x2x2xf32>{
+ %2 = tensor.pack %0  outer_dims_perm = [0, 3, 1, 2] inner_dims_pos = [3] inner_tiles = [2] into %1 : tensor<1x2x2x4xf32> -> tensor<1x2x2x2x2xf32>
```

Comment:
Would be good to have these tests checking IR here and the execution in `Integration`, so that we can make sure that it's not only doing what we want, but how we want.